src/Compilers/CSharp/Test/Syntax/Parsing/RefStructs.cs

```diff
@@ -49,12 +49,12 @@ class Program
 
             var comp = CreateCompilationWithMscorlib45(text, parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp7), options: TestOptions.DebugDll);
```

Comment:
Will use in one case - just for variety. 
In any case it is still clear from the error that we demand 7.2